committers.md

```diff
@@ -127,13 +127,41 @@ Git history for that code to see who reviewed patches before. You can do this us
 Changes pushed to the master branch on Apache cannot be removed; that is, we can't force-push to 
 it. So please don't add any test commits or anything like that, only real patches.
 
-All merges should be done using the 
-[dev/merge_spark_pr.py](https://github.com/apache/spark/blob/master/dev/merge_spark_pr.py) 
-script, which squashes the pull request's changes into one commit. To use this script, you 
+<h4>Setting up Remotes</h4>
+
+To use the `merge_spark_pr.py` script described below, you 
 will need to add a git remote called `apache` at `https://github.com/apache/spark`, 
-as well as one called "apache-github" at `git://github.com/apache/spark`. For the `apache` repo, 
-you can authenticate using your ASF username and password. Ask `dev@spark.apache.org` if you have trouble with 
-this or want help doing your first merge.
+as well as one called `apache-github` at `git://github.com/apache/spark`.
+
+You will likely also have a remote `origin` pointing to your fork of Spark, and
+`upstream` pointing to the `apache/spark` GitHub repo. 
+
+If correct, your `git remote -v` should look like:
+
+```
+apache	https://github.com/apache/spark-website.git (fetch)
+apache	https://github.com/apache/spark-website.git (push)
+apache-github	git://github.com/apache/spark-website (fetch)
+apache-github	git://github.com/apache/spark-website (push)
+origin	https://github.com/[your username]/spark-website.git (fetch)
+origin	https://github.com/[your username]/spark-website.git (push)
+upstream	https://github.com/apache/spark-website.git (fetch)
+upstream	https://github.com/apache/spark-website.git (push)
```

Comment:
In this context, these should be `spark.git` instead of `spark-website.git`.

Reply:
Oops yeah ran this in the wrong directory